src/components/ResultsMap.vue

```diff
@@ -1,5 +1,7 @@
 <template>
 	<section>
+		<results-list-item :result="this.selectedMarker"></results-list-item>
+		<p>Zoom out on map to ensure all result markers are visible.</p>
```

Comment:
The user feedback for the user to help them through the App helps a lot. Thank you for doing that!

Reply:
Anytime. Anything for some good UX!

Reply:
Thanks for moving this to the top! I forgot that we had talked about this UX change!